UnityProject/Assets/Scripts/Systems/Inventory/Pickupable.cs

```diff
@@ -167,6 +167,9 @@ private IEnumerator serverPreformInteractionLogic(HandApply interaction)
 
 		LeanTween.scale(gameObject, new Vector3(1, 1), 0.1f);
 
+		//Start the animation on all clients.
+		StartCoroutine(PickupAnim(interaction.Performer.gameObject));
```

Comment:
Might be better to move this to before the server animation as well

Reply:
Did you test it?